### PR TITLE
Test case for dataworks-kafka-producer-app

### PR DESCRIPTION
**Pull request by vipinchadha**

### CODE REVIEW

File: src/helpers/aws_helper.py [hunk 1]
```diff
@@ -1846,3 +1848,47 @@ def execute_commands_on_ec2_by_tags_and_wait(
 
     console_printer.print_info(f"Response from ssm {resp}")
     time.sleep(timeout)
+
+
+def purge_sqs_queue(queue_name, aws_region="eu-west-2"):
+    console_printer.print_info(f"Purging queue: {queue_name}")
+    service_name = "sqs"
+    client = boto3.client(service_name, region_name=aws_region)
```
Comment (steveburton4) [inline src/helpers/aws_helper.py:1856]: You should use the `get_client` method in this helper. It allows us to standardise all boto3 clients. Same for the methods below.

File: src/helpers/aws_helper.py [hunk 2]
```diff
@@ -1846,3 +1848,47 @@ def execute_commands_on_ec2_by_tags_and_wait(
...
+    )
+    time.sleep(10)
+
+    command_id = response["Command"]["CommandId"]
+    output = client.get_command_invocation(InstanceId=instance_id, CommandId=command_id)
+
+    return output
+
+
+def upload_file_to_s3(bucket, object_key, metadata, body, aws_region="eu-west-2"):
```
Comment (steveburton4) [inline src/helpers/aws_helper.py:1881]: Is there not a method that already does this? I'd be surprised if not..

**Review by tobywalls — DISMISSED** [review]

File: src/helpers/aws_helper.py (hunk 1 above)
Comment (vipinchadha) [reply src/helpers/aws_helper.py:1856]: Thanks Steve. changes applied.

File: src/helpers/aws_helper.py (hunk 2 above)
Comment (vipinchadha) [reply src/helpers/aws_helper.py]: Thanks Steve. changes applied.

**Review by steveburton4 — APPROVED** [review]

**Pull request merged**